src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -62,6 +62,10 @@
  */
 public class TableMetrics
 {
+    /**
+     * stores metrics that will be rolled into a single global metric
+     */
+    private final static ConcurrentMap<String, Set<Metric>> ALL_TABLE_METRICS = Maps.newConcurrentMap();
```

Comment:
renamed to all caps so its more clear this is global and not local to the table; seems that this naming convention caused `all` to be confused with local table names rather than global metrics, so switch to java naming convention to help to prevent this next time.